test/test_encodings.rb

```diff
@@ -0,0 +1,59 @@
+require File.dirname(__FILE__) + '/helper'
+
+class TestEncodings < Test::Unit::TestCase
+  def setup
+    @internal_encoding = Encoding.default_internal
+    @dir = Dir.mktmpdir("grit-test")
+    `cd "#{@dir}" && git  init`
+    @r = Repo.new(@dir)
```

Comment:
This was the pattern used in *most* of the existing tests. I was sticking with the dominant pattern for consistency.